CHANGES.md

```diff
@@ -1,3 +1,10 @@
+2.4.0 (2019-05-17)
+
+* UTop: depend on lambda-term 2 and therefore provide better unicode support(wide, combined glyph).
+* UTop\_lexer: support combined glyph
```

Comment:
I would suggest to only mention the parts that are public in the changelog. `UTop_lexer` is an internal module. The role of the changelog is to give users an overview of changes that might affect them.

So for instance, I would summarize the two above points into:

```
* Better support for wide and combined glyph (<your name or github id>, #NNN)
```

I just made up the sentence BTW, I'm not a unicode expert so I might have got it wrong. You can mention `UTop` if its API changed since it is a public module.